CHANGELOG.md

```diff
@@ -4,6 +4,7 @@ x.x.x Release notes (yyyy-MM-dd)
 * App crashed if a native error was thrown during `Realm.open(...)` ([#3414](https://github.com/realm/realm-js/issues/3414), since v10.0.0)
 * Fixed an issue in Node.js, where utilizing an ArrayBuffer for setting a binary property, would mangle the data. ([#3518](https://github.com/realm/realm-js/issues/3518))
 * Fixed an issue where scripts may hang rather than executing after all code has executed. ([#3525](https://github.com/realm/realm-js/issues/3518))
+* Fixed TS definitions for `Realm.ErrorCallback`.
```

Comment:
Haven't noticed this before, but it seems we're mixing the wording "TS definitions" & "TS declarations" ([typescriptlang.org](https://www.typescriptlang.org/docs/handbook/declaration-files/introduction.html) uses "declaration(s)") - should we streamline this? (maybe just going forward).

Reply:
I'll change it.